R/visualization_spatial.R

```diff
@@ -63,6 +68,9 @@ NULL
 #' visible in small plots.
 #' @param scalebar_position A numeric vector of length 2 specifying the position of the scale bar
 #' relative to the plot area. Default is to place it in the top right corner.
+#' @param tech Relevant for tile and raster shapes. A string indicating which technology the data comes from. 
+#' Defaults to \code{NULL} and will detect the technology automatically from the data. 
+#' Possible values are \code{c("vis", "vishd")}
```

Comment:
Might be a good idea to spell these out for clarity. E.g. "visium" and "visiumhd"

Reply:
Done. I have checked the code properly and there was no real need for a `tech` parameter. Thus, it has been removed and code related to it aswell